src/routes/news/actions/finishFetchMoreAction.js

```diff
@@ -0,0 +1,14 @@
+// @flow
```

Comment:
Should not be part of the route, but probably rather in module > app > actions

Reply:
It depends whether this action is used from multiple routes. If it is only used within routes/news/ then this is fine.

Reply:
Yeah I agree. It could be used on other routes. I will move it to module > app > actions